src/sage/graphs/base/boost_graph.pyx

```diff
@@ -1071,7 +1071,7 @@ cdef get_predecessors(BoostWeightedGraph g, result, int_to_v, directed, weight_t
         sage: johnson_shortest_paths(g, distances=False, predecessors=True) == expected
         True
     """
-    cdef vector[pair[int, pair[int, double]]] edges
+    cdef vector[pair[v_index, pair[v_index, double]]] edges
```

Comment:
This one and the above looks like good idea (matches the existing declaration in `boost_interface.cpp`)